bundles/org.openhab.binding.doorbird/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="doorbird" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="d101">
+		<label>Doorbird D101</label>
+		<description>Doorbird doorbell model D101</description>
+		<channels>
+			<channel id="doorbell" typeId="doorbell" />
+			<channel id="doorbellTimestamp" typeId="doorbellTimestamp" />
+			<channel id="doorbellImage" typeId="doorbellImage" />
+			<channel id="motion" typeId="motion" />
```

Comment:
We can use system default channel type `trigger` for this channel.

Reply:
Outdated: see https://github.com/openhab/openhab2-addons/pull/5857/files#r331783379